From 16a29892a94a6addcf6817c2e1256d9fd4e4dfb9 Mon Sep 17 00:00:00 2001
From: Jonghwa Lee <jonghwa3.lee@samsung.com>
Date: Wed, 19 Mar 2014 00:41:20 +0900
Subject: [PATCH 1300/1302] Revert "extcon: max77693: Force using UART path for
 jig"

This reverts commit ec05e8de55043b87a1dafccec5477db9a6ecf9a1.
Upcoming patch will fix a bug thus all related workaournds are now purged.

Change-Id: I52b4456867dd887c7d29a4f485c952860c6dc45c
Signed-off-by: Jonghwa Lee <jonghwa3.lee@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 drivers/extcon/extcon-max77693.c | 14 +++-----------
 1 file changed, 3 insertions(+), 11 deletions(-)

diff --git a/drivers/extcon/extcon-max77693.c b/drivers/extcon/extcon-max77693.c
index 903c507..3cde41b4 100644
--- a/drivers/extcon/extcon-max77693.c
+++ b/drivers/extcon/extcon-max77693.c
@@ -293,18 +293,10 @@ static int max77693_muic_set_path(struct max77693_muic_info *info,
 		return ret;
 	}
 
-	if (attached) {
-		if (info->prev_cable_type ==
-		    MAX77693_MUIC_ADC_FACTORY_MODE_UART_OFF) {
-			/* if cable_type is jig, then force UART */
-			dev_info(info->dev, "For jig force using UART path\n");
-			ctrl1 = CONTROL1_SW_UART;
-		} else {
-			ctrl1 = val;
-		}
-	} else {
+	if (attached)
+		ctrl1 = val;
+	else
 		ctrl1 = CONTROL1_SW_OPEN;
-	}
 
 	ret = max77693_update_reg(info->max77693->regmap_muic,
 			MAX77693_MUIC_REG_CTRL1, ctrl1, COMP_SW_MASK);
-- 
1.8.3.2

